UpdateTemplate.cmd

```diff
@@ -0,0 +1,27 @@
+@ECHO OFF
+PUSHD %~dp0
+
+SETLOCAL
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+:: Check if node exists globally
+WHERE node >NUL
+IF NOT '%ERRORLEVEL%'=='0' (
+    ECHO ERROR: UpdateTemplate.cmd requires node installed globally.
+    GOTO :Exit
+)
+
+SET TemplateHome=%~dp0src\docfx.website.themes\
+SET DefaultTemplate=%TemplateHome%default
+
+CALL npm install -g gulp
+
+PUSHD %DefaultTemplate%
+CALL npm install
+CALL gulp
+POPD
+
+PUSHD %TemplateHome%
+CALL npm install
+CALL gulp
+POPD
```

Comment:
actually only one pair of pushd and popd is needed at beginning and end of the script.

---
In reply to: [66747755](https://github.com/dotnet/docfx/pull/379#discussion_r66747755) [](ancestors = 66747755)